CONTRIBUTING.md

```diff
@@ -107,7 +97,7 @@ git rebase develop
 
 You should solve the merge conflicts, if any.
 
-9. **Pushing your code**
+1. **Pushing your code**
```

Comment:
Why did you make it 1

Reply:
Oh my bad! Mistake in copy paste, will fix